README.md

```diff
@@ -8,26 +8,45 @@ This project serves a variety of purposes including managing task assignments, f
-  - Linux: install with your favorite package manager. If mkcert is not available using your favorite package manager, run the following:
+-  [Docker Desktop](https://www.docker.com/products/docker-desktop)
+-  mkcert
+   -  Mac: install with [Brew](https://brew.sh) `brew install mkcert nss`
+   -  Windows: install with [Chocolatey](https://chocolatey.org) `choco install mkcert`
+   -  Linux: install with your favorite package manager. If mkcert is not available using your favorite package manager, run the following:
     ```
     curl -JLO "https://dl.filippo.io/mkcert/latest?for=linux/amd64"
     chmod +x mkcert-v*-linux-amd64
     sudo cp mkcert-v*-linux-amd64 /usr/local/bin/mkcert
     ```
-- [Stripe CLI](https://stripe.com/docs/stripe-cli)
+-  Set up [Stripe CLI](https://stripe.com/docs/stripe-cli)
+   -  Go to stripe.com and create an account.
+   -  When you are logged in, and at the dashboard page, click the link: API Keys for developers.
+   -  Here you will see your Public(Publishable) key and Private(Secret) key. Place these in your `.env` file.
+   -  Make sure you complete Setup > Step 6. 
+      - In the process you will get a pairing code, which you can ignore, continue by pressing `Enter` to open a browser and login to Stripe to continue the pairing process.
+   -  After running:
+      ```
+         stripe listen --forward-to https://localhost:8000/api/1/order/webhook
+      ```
+      you will get a webhook signing secret starting with `whsec_`, which you will place into your `.env` file as the [PRIVATE_STRIPE_WEBHOOK] variable. 
 
 ## Setup
 1. Clone the repository.
-2. Copy `.env.dist` to `.env`
-   1. Set the values for `AUTH0_CLIENT_ID` and `AUTH0_CLIENT_SECRET`. *you can get these values from the team lead*
-   2. Set the value for `PRIVATE_STRIPE_KEY` and `PUBLIC_STRIPE_KEY`. *you can get this value from the team lead*
-   3. Set the value for `PRIVATE_STRIPE_WEBHOOK`. *explained in step 5*
-3. In `<path/to/WonderTix/server>` Run `mkcert -install` to install the local certificate authority
-   1. Run `mkcert localhost` to create a certfiicate.   
+   1. Open your command line. 
+   2. Navigate to desired folder to install WonderTix repository
+   3. Execute the following command:
+      ```
+      git clone https://github.com/WonderTix/WonderTix.git
+      ```
+
+2. Create a `.env` file and copy over the contents from the `.env.dist` (.env example) file
+   1. Set the values for `AUTH0_CLIENT_ID` and `AUTH0_CLIENT_SECRET`. *you must get these values from the team lead*
+   2. Set the value for `PRIVATE_STRIPE_KEY` and `PUBLIC_STRIPE_KEY`. *you must get this value from the team lead*
+   3. Set the value for `PRIVATE_STRIPE_WEBHOOK`. **explained in step 5**
+3. Create mkcert certificate
+   1. Navigate to `<path/to/WonderTix/server>` 
+   2. Run `mkcert -install` to install the local certificate authority
+   3. Run `mkcert localhost` to create a certfiicate.   
 5. Run `docker-compose up -d`
```

Comment:
Although the README renders correctly, could you explicitly label this as `4.` instead of `5.` and continue the count forward

Reply:
Good catch!

Reply:
fixed